### PR TITLE
Edu 4587 stock keeping unit by sales channel route

### PR DESCRIPTION
**Pull request by karenkrieger**
#### Types of changes
- [X] New content (endpoints, descriptions or fields from scratch)
- [ ] Improvement (make an endpoint's title or description even better)
- [ ] Spelling and grammar accuracy (self-explanatory)

New query param

### CODE REVIEW

File: VTEX - Catalog API.json [hunk 1]
```diff
@@ -20900,6 +20900,16 @@
                             "type": "integer",
                             "default": "1"
                         }
+                    },
+                    {
+                        "name": "onlyAssigned",
+                        "in": "query",
+                        "required": false,
+                        "description": "If set as `true`, it allows user to decide if the SKUs that are not assigned to a specific trade policy should be also returned",
```
Comment (vilmabezerra) [inline VTEX - Catalog API.json:20908]: Actually, if set as `false` the SKUs not assigned to a specific trade policy should be also returned.

File: VTEX - Catalog API.json [hunk 2]
```diff
@@ -20900,6 +20900,16 @@
...
+                        "schema": {
+                            "type": "boolean",
+                            "default": false
```
Comment (vilmabezerra) [inline VTEX - Catalog API.json:20911]: the default behaviour is set as `true`

**Review by vilmabezerra — CHANGES_REQUESTED** [review]
Hey, it looks awesome, thanks! 
Only a specific detail, that I wasn't clear about it before, that needs to be changed.

**Review by brunoamui — APPROVED** [review]
LGTM

**Pull request merged**